rlel/Account.xaml.cs

```diff
@@ -163,5 +191,17 @@ public partial class Account : UserControl {
         private void UserControl_MouseDoubleClick(object sender, System.Windows.Input.MouseButtonEventArgs e) {
             this.launchAccount();
         }
+
+        public bool ClientIsRunning()
+        {
+            if (this.main.singularity.IsChecked == true && this.tranqProcess != null)
+                return !this.tranqProcess.HasExited;
+            else if (this.main.singularity.IsChecked == false && this.sisiProcess != null)
+                return !this.tranqProcess.HasExited;
```

Comment:
1. should the first condition be checking `tranqProcess`?
2. should the first return be checking `tranqProcess`?
3. should the second condition be checking `sisiProcess`?
4. please use `if (some_bool)` and `if (!some_bool)` instead of testing with `==`